reader/processor/processor.go

```diff
@@ -120,6 +133,39 @@ func ProcessEntryWebPage(entry *model.Entry) error {
 	return nil
 }
 
+func fetchYouTubeWatchTime(url string) int {
+	clt := client.NewClientWithConfig(url, config.Opts)
+	response, browserErr := browser.Exec(clt)
+	if browserErr != nil {
+		return 0
+	}
+
+	doc, docErr := goquery.NewDocumentFromReader(response.Body)
+	if docErr != nil {
+		return 0
+	}
+
+	durs, exists := doc.Find(`meta[itemprop="duration"]`).First().Attr("content")
+	if !exists {
+		return 0
+	}
+
+	dur, err := duration.ParseISO8601(durs)
+	if err != nil {
+		return 0
+	}
+
+	return int(timeDuration(dur).Minutes())
```

Comment:
Do you mind adding some unit test? at least for the parsing part. Just to make sure durations like `PT21M4S` or `PT0M0S` are correctly parsed.